src/Chapter1.hs

```diff
@@ -490,7 +491,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit = (`mod` 10) . abs
```

Comment:
That's pretty!

Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: